task/sast-snyk-check-oci-ta/0.4/sast-snyk-check-oci-ta.yaml

```diff
@@ -228,8 +228,14 @@ spec:
           snyk code test $ARGS --severity-threshold="$SEVERITY_THRESHOLD" "$resolved_path" --max-depth=1 --sarif-file-output="${resolved_path}/sast_snyk_check_out_${d//\//_}.json" 1>&2 >>stdout.txt || true
         done
 
-        # Merge all SARIF outputs
-        find "$SOURCE_CODE_DIR" -name "sast_snyk_check_out_*.json" -exec cat {} + >"${SOURCE_CODE_DIR}/sast_snyk_check_out.json"
```

Comment:
I do not understand how you can merge JSON files with `cat`.  That will not produce a valid JSON file.  This problem was already introduced in commit 244f032aa70f2d1d5fd1b3f2fe033b5c22a69d48.

Reply:
That is a great point, not sure how that was missed earlier.. we will need to fix.

Reply:
> I'll fix using jq

I do not think this would be easy:

- If you get valid JSON, it does not mean that you get valid SARIF.
- If you get valid SARIF, it does not mean that `csgrep` will be able to process it correctly (because `csgrep` implements only a small subset of the SARIF specification).

I would suggest to rather use `csgrep` directly to read all the SARIF files you need to process.